src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,55 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: ($day-size + $gap) * $col-count - $gap;
```

Comment:
Ensure that the variables $day-size, $gap, and $col-count are defined elsewhere in your SCSS files. If they are not defined, this will cause a compilation error.